client/app/stores/message_store.coffee

```diff
@@ -147,14 +144,16 @@ class MessageStore extends Store
                     [key, value] = p.split '='
                     value = '-' if value is ''
                     _params[key] = value
-            else
+            else if messages.mailboxID
```

Comment:
The infinite scroll breaks after an action, because we use RECEIVE_RAW_MESSAGES there. I thought it was due to my changes and made the fix. Then found out it was already there in #271